aardwolf-templates/templates/widgets/icon.rs.html

```diff
@@ -0,0 +1,37 @@
+/// Icon Shortcut Defintions
+/// The list below reduces the code for setting icons in the templates.
+///
+///  Example Usage:
+///  `@icon:"github"`
+///
+/// The example above will be compiled as:
+/// `<span class="fa fa-github"></span>`
+/// 
+/// It is also possible to use the five other sizes provided by FontAwesome
+/// ```
+///  @icon_lg:"Github"  // 33% larger
+///  @icon_2x:"Github"  // 2x larger
+///  @icon_3x:"Github"  // 3x larger
+///  @icon_4x:"Github"  // 4x larger
+///  @icon_5x:"Github"  // 5x larger
+/// 
+/// For more info on the icon sizes see the link below
+/// Source: https://forkawesome.github.io/Fork-Awesome/examples/
+
+@(icon: &str)
```

Comment:
this file doesn't make any sense. Each template file can only have one template in it.

try making multiple template files like
`icon.rs.html`, `icon_2x.rs.html`, `icon_3x.rs.html`, etc.

Reply:
So we will need to have one file for each set of icon size definitions?